scene/main/node.cpp

```diff
@@ -2485,6 +2485,12 @@ void Node::replace_by(Node* p_node,bool p_keep_data) {
 			rd.name=E->get().name;
 			rd.value=get(rd.name);
```

Comment:
**Not related to this PR**: It looks like this block of code is not finished. Here `rd` should be added to `replace_data`, shouldn't it?